plugins/advisors/black-duck/src/funTest/kotlin/ResponseCachingComponentServiceClient.kt

```diff
@@ -40,7 +40,7 @@ internal class ResponseCachingComponentServiceClient(
     private val serverUrl: String?,
     apiToken: String?
 ) : ComponentServiceClient {
-    // The BlackDuck library uses GSON to serialize its POJOs. So use GSON, too, because this is the simplest option.
+    // The Black Duck library uses GSON to serialize its POJOs. So use GSON, too, because this is the simplest option.
```

Comment:
Commit message nit: Some Hint / reference about the official spelling would have been nice, like linking to https://www.blackduck.com/company.html. Because interestingly, while in the logo it looks like one word, in prose text they spell themselves indeed as "Black Duck".